pkg/bpf/config.go

```diff
@@ -36,18 +36,20 @@ func init() {
 }
 
 type Config struct {
-	BpfFsPath        string `json:"-bpf-fs-path"`
-	Cgroup2Path      string `json:"-cgroup2-path"`
-	EnableKmesh      bool   `json:"-enable-kmesh"`
-	EnableMda        bool   `json:"-enable-mda"`
-	BpfVerifyLogSize int    `json:"-bpf-verify-log-size"`
+	BpfFsPath           string `json:"-bpf-fs-path"`
+	Cgroup2Path         string `json:"-cgroup2-path"`
+	EnableKmesh         bool   `json:"-enable-kmesh"`
+	EnableKmeshWorkload bool   `json:"-enable-kmesh-workload"`
```

Comment:
The xDS type and workload type cannot be enabled at the same time and need to be isolated. Check whether isolation can be implemented using commands, for example, -enable-kmesh -enable-workload=true/-enable-ads=true.